doc/datasets/olivetti_faces.rst

```diff
@@ -11,7 +11,7 @@ This dataset contains a set of face images taken between April 1992 and April
 fetching / caching function that downloads the data
 archive from AT&T.
 
-.. _This dataset contains a set of face images: http://www.cl.cam.ac.uk/research/dtg/attarchive/facedatabase.html
+.. _This dataset contains a set of face images_: http://www.cl.cam.ac.uk/research/dtg/attarchive/facedatabase.html
 
```

Comment:
Sorry I was unclear. Remove the_ you added in the link, but put it in the text in the paragraph above. Model it after the previous link that was there (note the markup around the "Internet way back machine" text that you removed in the diff) 
I'd encourage you to build the docs by running `make html-noplot `in the docs directory and seeing if the generated html looks right.